code/datums/hud.dm

```diff
@@ -1,7 +1,7 @@
 /* HUD DATUMS */
 
 //GLOBAL HUD LIST
-var/datum/atom_hud/huds = list( \
+var/list/datum/atom_hud/huds = list( \
```

Comment:
While coding,i've seen that this list isn't declared as a list. That made list procs not work such as list.Add(stuff) and other things,so i've added it, just to be clear